egs2/how2/sum1/local/run_asr.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
it would be better to move this to `egs2/how2/asr1/run.sh`.
You can do it in your follow up PR, later.

Reply:
One issue might be that the current recipe is for How2-300h, while my run_asr operates on How2-2000h

Reply:
how about making a new directory then?
how2: 300h
how2_2000h: 2000h

Reply:
Will do in a future PR